setup.py

```diff
@@ -51,6 +51,6 @@
         # Include any JSON schemas.
         '': ['*.json'],
     },
-    python_requires='>=3.7',
+    python_requires='>=3.10',
```

Comment:
https://github.com/google/osv.dev/blob/master/CONTRIBUTING.md#prerequisites states Python 3.11, so for consistency we might want to make this that...

```suggestion
    python_requires='>=3.11',
```

Reply:
Good point, thanks Andrew!